src/index.tsx

```diff
@@ -302,28 +300,28 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
         toValue: alwaysOpen && dest === 'default' ? 0 : 1,
         duration: timing.duration,
         easing: Easing.ease,
-        useNativeDriver,
+        useNativeDriver: useNativeDriver!,
```

Comment:
Since we are using a default value for `useNativeDriver` in the constructor, you can simply remove the optional `?` from the option.ts file